build.gradle

```diff
@@ -180,6 +186,7 @@ dependencies {
 
     compileOnly group: 'org.jetbrains', name: 'annotations', version:'17.0.0'
     compileOnly group: 'com.yuvimasory', name: 'orange-extensions', version: '1.3.0'
+    compileOnly group: 'com.github.spotbugs', name: 'spotbugs-annotations', version: '3.1.12'
```

Comment:
This seems out of place in a browser swap...

Reply:
I was going for "all new gui3 code will use the new rules..." but yeah, this should be a different, followup, change.